projects/storefrontapp-e2e-cypress/cypress/helpers/asm.ts

```diff
@@ -175,4 +213,46 @@ export function testCustomerEmulation() {
     navigateToCategory('Brands', 'brands', false);
     cy.get('cx-product-list-item').should('exist');
   });
+
+  it('should verify data changed by the agent as a customer', () => {
```

Comment:
included here because I believe it's a core e2e because without it, how would we know it actually worked